source/mindmeld_ui/mindmeld_ui.rst

```diff
@@ -0,0 +1,41 @@
+MindMeld UI
+===========
+
+Included in the `mindmeld` repository there is also a web application that visually demonstrates how the platform works. This is a React application that can be run along with any of the blueprints, or any `mindmeld` based app.
+
+We recommend running this app with a recent version of Chrome to guarantee access to the ASR engine.
+
+Quick Start
+-----------
+
+You can run this app by following the next steps while inside the `mindmeld-ui` folder of the `mindmeld` repository.
+
+1. Install the dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order to run the app, you need to install it's dependencies. You can simply run:
+
+.. code-block:: console
+
+    npm install
+
+
+2. Run the app
+^^^^^^^^^^^^^^
+
+To run the app simply do:
+
+.. code-block:: console
+
+    npm start
+
+3. Setup configurations (Optional)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+At the `mindmeld-ui` directory, you can add a `.env` file that includes your configurations. This file should look like this:
```

Comment:
"In the `mindmeld-ui` directory, you can add an optional `.env` file with extended configuration. This file can include the following options:"

You could also include an example .env file in the repo called `.env-example` or something similar.

Reply:
Added.